docs/content/python/guides/enforcement/field.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Field-level Enforcement"
+weight: 30
+any: true
+description: >
+  Learn about enforcing field-level authorization, controlling who can access
+  which fields of your resources.
+# draft: True
+---
+
+{{% callout "Note: 0.20.0 Alpha Feature" %}}
+  This is an API provided by the alpha release of Oso 0.20.0, meaning that it is
+  not yet officially released. You may find other docs that conflict with the
+  guidance here, so proceed at your own risk! If you have any questions, don't
+  hesitate to [reach out to us on Slack](https://join-slack.osohq.com). We're
+  here to help.
+{{% /callout %}}
+
+<div class="pb-10"></div>
+
+# Field-level Enforcement
+
+Perhaps you're creating a user profile page, and in your app, only a user's
+friends can access their last check-in location. To enforce this type of
+authorization, it often makes sense to use **"field-level" enforcement**, by
+explicitly allowing access to certain fields on your domain objects.
```

Comment:
How do I know whether to use `field-level enforcement` or `resource-level enforcement`?

In this example, I could write a rule over the Location itself and only allow users to access the location if it's their own, or they're in the friends list of the user for the location.

Reply:
I'll admit that I was hoping to avoid answering this question directly because it is a bit subtle. In general I would worry more about people using field-level rules where resource-level rules make more sense (as in this `Location` example). 

Perhaps an example with the `email` field would be better? It might more clearly suggest that it is about "fields on this object" instead of "objects related to this object".

Reply:
Yeah I think that example would bring up this question less.